### ISSUE TITLE
Pause after current track button

### ISSUE THREAD

**Issue by azumukupoe** [nukeop/nuclear#1103]
It would be nice to have a button to stop playback on finishing current track

**Comment by nukeop** [comment]
Is there precedence for this option in another music player?

**Comment by HaiDang666** [comment]
I don't think this is a practical case. If your case is stop looking new track, you can turn off autoradio. Or we can implement `Sleep timer` function instead.

**Comment by azumukupoe** [comment]
> Is there precedence for this option in another music player?

I think Clementine has this feature

**Comment by nukeop** [comment]
This issue is from an older version of Nuclear that has been replaced with a complete rewrite. None of the old issues apply to the new codebase.

The old code is preserved on the [legacy/electron](https://github.com/nukeop/nuclear/tree/legacy/electron) branch. If you have feedback about the new version, head to [Discussions](https://github.com/nukeop/nuclear/discussions).
